src/warnet/bitcoin.py

```diff
@@ -39,10 +38,19 @@ def _rpc(tank: str, method: str, params: str):
     # bitcoin-cli should be able to read bitcoin.conf inside the container
     # so no extra args like port, chain, username or password are needed
     namespace = get_default_namespace()
+
+    try:
+        pod = get_pod(tank)
+        container_names = [container.name for container in pod.spec.containers]
+        container_name = container_names[0]
```

Comment:
What determines the order of containers in a pod? Can we guarantee the one we want is always `[0]` because of the yaml file? Or should we hard code the name from the chart `bitcoincore` ?